### PR TITLE
 update: readme

### PR DESCRIPTION
**Pull request by ValleyZw**
Add 1 documentary and 2 youtube channels

### CODE REVIEW

**Comment by nikivdev** [comment]
Hey sorry I didn't see the PR.

Looks good although is 3blue1brown a documentary? It's a youtube channel.

File: README.md [hunk 1]
```diff
@@ -86,12 +88,15 @@
...
 - [Touching the Void](https://letterboxd.com/film/touching-the-void/)
 - [Meru](https://letterboxd.com/film/meru/)
+- [The Act Of Killing](https://www.youtube.com/watch?v=3tILiqotj7Y)
 
 ## Related
 
 - [Documentary heaven](http://documentaryheaven.com/)
 - [Documentary storm](https://documentarystorm.com/)
 - [Top documentary films](http://topdocumentaryfilms.com/)
+- [LEMMiNO](https://www.youtube.com/user/Top10Memes)
```
Comment (nikivdev) [inline README.md:98]: not a doc

File: README.md [hunk 2]
```diff
@@ -86,12 +88,15 @@
...
+- [3Blue1Brown](https://www.youtube.com/channel/UCYO_jab_esuFRV4b17AJtAw)
```
Comment (nikivdev) [inline README.md:99]: not a doc

File: README.md [hunk 3]
```diff
@@ -28,6 +28,7 @@
 
 ## Earth
 
+- [An Inconvenient Truth](https://letterboxd.com/film/an-inconvenient-truth/)
```
Comment (nikivdev) [inline README.md:31]: doesn't have good rating

**Comment by ValleyZw** [comment]
Oh, sorry, I think 3b1b's works are fabulous, can't help to mention the channel.

**Comment by nikivdev** [comment]
Please change these & I will merge

**Comment by ValleyZw** [comment]
ok

**Pull request closed without merging**